hugo/layouts/projects/single.html

```diff
@@ -5,6 +5,18 @@
 <body class="projects-page">
     {{ partial "header.html" $headerCtx }}
     <section class="projects-hero">
```

Comment:
it can't be reused, it has many different things. It has an image, the other has a slack form, this has one code pills image, the other has two. This title is with a pill, the other without. The only thing they have in common is they have a title, a description and other stuff. Not sure an abstraction over this is better than a small duplication.